src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+  </Side>
+  </SideBySide>
+
+# Service levels [#service-levels]
+<SideBySide>
+  <Side>
+
+  With New Relic, you can set and use service level indicators (SLIs) and objectives (SLOs) for your applications. Service levels measure the performance of a service from the end user (or client application) point of view. We measure service levels with indicators, comparing them to set objectives for performance and outcomes. 
+
+  Learn more about service levels [here](/docs/service-level-management/intro-slm/).
+
+  </Side>
+  <Side>
+
+  <img
+  title="New Relic Service levelss"
+  alt="New Relic Service levels"
+  src={NRServiceLevelsMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Synthetic monitoring [#synthetic]
+<SideBySide>
+  <Side>
+
+<img
+  title="New Relic synthetic"
+  alt="New Relic synthetic"
+  src={NRSyntheticMain}
+/>
+
+  </Side>
+  <Side>
+
+The <InlinePopover type="synthetics" /> capability lets you proactively check that your services are working. Use synthetic monitoring to find problems in your customer-facing apps. It lets you spot issues before they impact your users. Synthetic monitoring simulates user interactions to pinpoint errors. This proactive approach ensures prompt addressing of any issues. Stay ahead of potential problems by leveraging synthetic monitoring tools. 
+
+Learn more about synthetics [here](/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/).
+
+  </Side>
+  </SideBySide>
+
+# Vulnerability management [#vuln-management]
+<SideBySide>
+  <Side>
+
+  Vulnerability management provides a birds-eye view of all your software's vulnerabilities. Use our <InlinePopover type="vulnerabilityManagement" /> to find the most urgent vulnerabilities. Calculate the vulnerability surface area of your software and assign vulnerabilities to users with explicit steps for fixing them.
+
+  Learn more about vulnerability management [here](/docs/vulnerability-management/overview/).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic vulnerability management"
+    alt="New Relic vulnerability management"
+    src={NRVulnManagementMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+
+# How it works [#how-it-works]
+
+The first step in observability requires collecting lots of performance data, known as telemetry data. Our observability platform gathers data from your services using agents. An agent is a file or a few lines of code that collects data on your service and sends it to our platform for analysis. 
+
+A process called instrumentation integrates agents into a service. The agent keeps track of your service's performance by monitoring metrics such as response times and error rates. By analyzing this data, you can gain insights into the functioning of your service and identify areas for improvement. An agent is first installed in your environment with a default configuration. The process of installation, setup, and instrumentation can vary. It depends on the service you want to collect data from. 
```

Comment:
I think we can make this a little more high level / introduce a metaphor to describe what instrumentation actually is. Maybe something like:

> Instrumentation is the process of making something measurable. You might instrument a gas tank by installing a meter that measures how full or empty your tank is. You monitor your tank, then when it approaches empty, you know it’s time to fill up again. 
> 
> New Relic does something similar. An agent is a file or piece of code that you attach to a particular service during an install procedure. Like a gas meter, our agents instrument technologies and collect data about how that technology performs. You can use that telemetry data for analysis, letting you gain insight about opportunities for optimization and improvement. 
>

Reply:
Great idea!! I added it! Thanks!